apis/Google.Cloud.Diagnostics.AspNet/Google.Cloud.Diagnostics.AspNet/Trace/CloudTrace.cs

```diff
@@ -122,7 +122,7 @@ public static void Initialize(string projectId, HttpApplication application, Tra
 
         private void BeginRequest(object sender, EventArgs e)
         {
-            TraceHeaderContext headerContext = TraceHeaderContext.FromRequest(HttpContext.Current.Request);
+            TraceHeaderContext headerContext = TraceHeaderContextUtils.FromRequest(HttpContext.Current.Request);
```

Comment:
Nit: `FromRequest` no longer seems appropriate as a name. `TraceHeaderContextUtils.FromRequest` looks like it is creating a `TraceHeaderContextUtils`. Maybe rename to `CreateContext` or something else?